en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md

```diff
@@ -1,11 +1,11 @@
 # Importing APIs Via Dev First Approach
 
-WSO2 API Controller, **apictl** allows to create and deploy APIs without using WSO2 API Publisher Portal. You can use this feature to create an API **from scratch** or **using an existing Swagger or Open API specification** and then deploy it to the desired API Manager environment.
+WSO2 API Controller (**apictl**) allows you to create and deploy APIs without using WSO2 API Publisher Portal. You can use this feature to create an API **from scratch** or **using an existing Swagger or Open API specification** and then deploy it to the desired API Manager environment.
 
 !!! info
     **Before you begin** 
 
-    -   Make sure WSO2 API Manager CTL Tool is initialized and running, if not, follow the steps in [Download and Initialize the CTL Tool]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool).
+    -   Make sure that the WSO2 API Manager CTL Tool is initialized and running, if not, follow the steps in [Download and Initialize the CTL Tool]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool).
```

Comment:
_"Make sure that the WSO2 API Manager CTL Tool is initialized and **running**"_ , seems to communicate that the ctl needs to be running in the background before proceeding with the next steps.

Shall we change it to _"Make sure that the WSO2 API Manager CTL Tool is  downloaded and initialized"_ ?

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/2559/commits/708589ec269c76201da4254ec4d018824fa1cfd8